src/components/Card/Card.module.css

```diff
@@ -38,12 +39,14 @@
   display: flex;
   justify-content: space-between;
   padding: calc(var(--spacing-padding-sm, 8) * 1px) calc(var(--spacing-padding-xl, 24) * 1px);
+  gap: calc(var(--spacing-gap-xl, 24) * 1px);
 }
 
 .heading {
   display: flex;
   flex-direction: column;
   gap: calc(var(--spacing-gap-sm, 8) * 1px);
+  overflow-x: hidden;
```

Comment:
why we need to use these the overflow-x?

Reply:
We need it to prevent the title from breaking through the header and the card itself:

![Screenshot from 2024-02-20 13-31-57](https://github.com/mia-platform/design-system/assets/125553486/a7e49361-4a0a-41b1-92de-aede358e5ee8)